### PR TITLE
Feat/chfo/mergedylib6 compiler

### PR DESCRIPTION
**Pull request by cfogelklou**
This is a pull request for the compiler changes needed to be able to output *.dylib/*.so libraries built with robovm.

Glad to finally make these pull requests.  Hope they get accepted after all this time :)

### CODE REVIEW

**Comment by ntherning** [comment]
Thanks Chris! Will take a look and feed back to you next week.

File: compiler/src/main/java/org/robovm/compiler/config/Config.java [hunk 1]
```diff
@@ -1361,7 +1396,12 @@ public void addPluginArgument(String argName) {
             }
             config.pluginArguments.add(argName);
         }
-
+        
+        // TODO: Are we disobeying any rules by allowing config to be fetched?  All I want is the targetBinary.
```
Comment (badlogic) [inline compiler/src/main/java/org/robovm/compiler/config/Config.java]: I think i'd be better to have a getter in the builder. We do something similar for plugins imo.

Comment (cfogelklou) [inline compiler/src/main/java/org/robovm/compiler/config/Config.java]: Hi, 

Thanks, I will address this! 

Best Regards 

Chris Fogelklou, 
Applaud Apps (Applicaudia) 
Sent from my mobile, excuse the brevity. 

-------- Original Message --------

Comment (ntherning) [inline compiler/src/main/java/org/robovm/compiler/config/Config.java]: @cfogelklou I'm currently looking into merging the PRs. I will clean this up.

Comment (cfogelklou) [inline compiler/src/main/java/org/robovm/compiler/config/Config.java]: Do you need some help?  I just found the discussion in github - had looked several times but it wasn't easy to find!  I could rebase this branch to your latest master and then also fix this.

**Pull request closed without merging**